### ISSUE TITLE
Needs more Unsafe

### ISSUE THREAD

**Issue by tth05** [Conclure/DeluxeAsyncJoinLeaveMessage#4]
Creating an instance of a class calls its constructor, which is very very very extremely slow! I recommend creating all instances using the Unsafe class to ensure maximum performance.

**Comment by conclube** [comment]
Yep very wise issue my friend. I'm planning on making everything unsafe based but I still need as verbosely code as possible 🔥🔥

**Comment by tth05** [comment]
My prayers go out to the families that don't have maximum performance yet 🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏

**Comment by cumhilo** [comment]
How could no one have thought of this before? that it hurts people who create an instance of a class and call its constructor 😥🙏
